java/bfg/Portfile

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           java 1.0
+
+name                bfg
+version             1.13.0
+
+categories          java devel
+license             GPL-3.0
+maintainers         {@ra1nb0w irh.it:rainbow} openmaintainer
+platforms           darwin
+supported_archs     noarch
+
+description         Removes large or troublesome blobs like git-filter-branch does, but faster.
+long_description    The BFG is a simpler, faster alternative to git-filter-branch for cleansing \
+                    bad data out of your Git repository history: \n \
+                      - Removing Crazy Big Files \n \
+                      - Removing Passwords, Credentials & other Private data
+homepage            https://rtyley.github.io/bfg-repo-cleaner/
+
+master_sites        https://search.maven.org/remotecontent?filepath=com/madgag/${name}/${version}/
+distfiles           ${name}-${version}.jar
+distname            ${name}-${version}
+worksrcdir          ${name}-${version}
+
+checksums           rmd160  6f692650684b52f5f78801d57fddd2bc1d4d7595 \
+                    sha256  bf22bab9dd42d4682b490d6bc366afdad6c3da99f97521032d3be8ba7526c8ce \
+                    size    13465496
+
+depends_run         bin:java:kaffe
+java.version        1.7+
+
+use_configure       no
+build.cmd           true
+worksrcdir          .
```

Comment:
You have set the `worksrcdir` for a 2nd time here.

Reply:
ops. done